src/routes/api/cryptor-migrations/migration-installation-verification.ts

```diff
@@ -0,0 +1,39 @@
+import { Request, Response } from "express";
+import { Installation } from "models/installation";
+
+export const CryptorMigrationInstallationVerificationPost = async (req: Request, res: Response): Promise<any> => {
+
+	const jiraHost: string = req.body?.jiraHost;
+
+	req.log = req.log.child({ operation: "migrate-installations-verification" });
+
+	if (!jiraHost) {
+		res.status(200).json({ ok: false, reason: "invalid jiraHost param in body" });
+		return;
+	}
+
+	const inst: Installation = await Installation.findOne({ where: { jiraHost } });
```

Comment:
use the function that already exists for Installation: `Installation.getForHost(jiraHost)`

Reply:
fixed.